File: src/main/kotlin/helpers/Formatter.kt
package helpers

fun formatResponse(data: Map<String, Any>): Map<String, Any> {
    return data + mapOf("formatted" to true, "status" to 200)
}

fun formatError(err: String): Map<String, Any> {
    return mapOf("status" to 500, "error" to err)
}

fun formatMessage(msg: String): String {
    return "[MSG] $msg"
}

fun formatDate(timestamp: Long): String {
    return timestamp.toString()
}

File: src/Helpers/formatter.php
<?php

function format_response($data) {
    return ['status' => 200, 'body' => $data, 'formatted' => true];
}

function format_error($err) {
    return ['status' => 500, 'error' => $err];
}

function format_date($timestamp) {
    return date('Y-m-d', $timestamp);
}

function format_json($data) {
    return json_encode($data);
}
